package.json

```diff
@@ -21,6 +21,7 @@
     "test:esm:browser": "polendina --page --worker --serviceworker --cleanup dist/esm/browser-test/test-*.js",
     "test:ts": "npm run build:types && npm run test --prefix test/ts-use",
     "test": "npm run lint && npm run test:node && npm run test:esm && npm run test:ts",
+    "typecheck": "tsc --build test/tsconfig.json",
```

Comment:
Does it get run by the `gozala/typescript-error-reporter-action`? If not, then I don't think this gets run by anything automatically, or manually unless someone runs it directly, is it?

Let's wire this up to one of the main scripts that devs will be using- maybe just add it in to `"test"`, `"build:types"` would also work but it's not really a "build" script per se.

Reply:
I just mostly added for convenience so I could run `npm run typecheck`, it should not create build artifacts because `tsconfig.json` has `noEmit`. I wish there was `tsc check` but sadly TS is not exactly what some would wish for.

I can remove this